lib/unison-sqlite/package.yaml

```diff
@@ -0,0 +1,50 @@
+name: unison-sqlite
+github: unisonweb/unison
+copyright: Copyright (C) 2013-2021 Unison Computing, PBC and contributors
+
+library:
+  source-dirs: src
+  other-modules:
+    - Unison.Sqlite.DataVersion
+    - Unison.Sqlite.JournalMode
+    - Unison.Sqlite.Sql
+
+dependencies:
+  - base
+  - direct-sqlite
+  - exceptions
+  - mtl
+  - recover-rtti
```

Comment:
I'm not opposed to this library on principle, but I will note the following:

* It's less than a year old
* It's had 4 distinct major version releases in that year
* It's been downloaded only 332 times (at time of writing), so it's not exactly battle-tested.

For most libraries this would be enough to scare me off, in this case we're using it for a pretty non-critical task (just showing things for debug info), so it's not a big deal necessarily, but since it inspects Heap info I just don't want it to start failing on us in a future GHC release.

All that said, seems like unnecessary risk to effectively avoid a Show constraint. What do you think?

Reply:
Good points, but I'm still in favor of using the library. It's developed and maintained by Well Typed so I don't think it's likely to break, and if it does, it will be easy to remove.

I think it's pretty nice to not have superfluous `Show` constraints on the params we want to show, for debugging a broken query.

Reply:
Sure, we can always change it later if ends up causing us problems.